plugins/inputs/ethtool/README.md

```diff
@@ -0,0 +1,43 @@
+# Ethtool Input Plugin
+
+The ethtool input plugin pulls ethernet device stats.  Fields pulled will depend on the network device and driver
+
+
+
+### Configuration:
+
+```toml
+# Returns ethtool statistics for given interfaces
+[[inputs.ethtool]]
+  ## List of interfaces to pull metrics for
+  # include_interfaces = ["eth0"]
+  
+  ## List of interfaces to ignore when pulling metrics.  
+  # ignore_interfaces = ["lo0"]
+  
+  ## List of stat fields to include - default is all
+  # include_fields = [""]
+  
+  ## Add driver information as tag
+  driver_name_tag = true
+```
+
+Interfaces can be included or ignored using 
+
+- `include_interfaces`
+- `ignore_interfaces`
+
+Fields can be filtered using the `include_fields` array
+
+The driver name can be dropped by setting `driver_name_tag` to false
+
+### Metrics:
+
+Metrics are dependant on the network device and drive
+
+### Example Output:
+
+```
+1558004820000000000 sfc    host1 sfc0      34503285             17460420127        2110321               2864308103         575875087           461432982  1936943999        1331        330765            0                   83297760          5031166659864 0               19816863653                   1916                 5423                1                             0                            23335592573  5031166329099      0               0            21155071896       467                  0                23335593904     0             0                              0                          0                      118510                        0                            0                    0                           2097222917      5935186              37589985           19562047              1970004687         65731417            1337584    11020953          15                769847731426  0               0            104088            2111181859      4             2111077752                                                                                                                                                                                                                                                                                                          33185072        386225          367805          364416                          0              0            0                          0                           0                    0                 0               0                0               0              0                          0                           0        0                     0            95859           82516           61007           82432                           1             1982            241268         321814    0             0                0                   0              server
```

Comment:
Update this to be the output run with the example config (producing influx line protocol):

```toml
[[outputs.file]]

[[inputs.ethtool]]
```

Which currently gives the following (this is from an interface with most fields, though it doesn't have to be):
```
ethtool,driver=e1000e,interface=enp0s0f0 alloc_rx_buff_failed=0i,collisions=0i,corr_ecc_errors=0i,dropped_smbus=0i,multicast=0i,rx_align_errors=0i,rx_broadcast=0i,rx_bytes=0i,rx_crc_errors=0i,rx_csum_offload_errors=0i,rx_csum_offload_good=0i,rx_dma_failed=0i,rx_errors=0i,rx_flow_control_xoff=0i,rx_flow_control_xon=0i,rx_frame_errors=0i,rx_header_split=0i,rx_hwtstamp_cleared=0i,rx_length_errors=0i,rx_long_length_errors=0i,rx_missed_errors=0i,rx_multicast=0i,rx_no_buffer_count=0i,rx_over_errors=0i,rx_packets=0i,rx_short_length_errors=0i,rx_smbus=0i,tx_abort_late_coll=0i,tx_aborted_errors=0i,tx_broadcast=0i,tx_bytes=0i,tx_carrier_errors=0i,tx_deferred_ok=0i,tx_dma_failed=0i,tx_dropped=0i,tx_errors=0i,tx_fifo_errors=0i,tx_flow_control_xoff=0i,tx_flow_control_xon=0i,tx_heartbeat_errors=0i,tx_hwtstamp_skipped=0i,tx_hwtstamp_timeouts=0i,tx_multi_coll_ok=0i,tx_multicast=0i,tx_packets=0i,tx_restart_queue=0i,tx_single_coll_ok=0i,tx_smbus=0i,tx_tcp_seg_failed=0i,tx_tcp_seg_good=0i,tx_timeout_count=0i,tx_window_errors=0i,uncorr_ecc_errors=0i 1563904397000000000
```

Reply:
updated to InfluxData format.